### DISCUSSION TITLE
[Feature]  MOE多模态模型 许愿

### DISCUSSION THREAD

**Discussion by sanbuphy** [InternLM/InternLM#671] [Ideas]
### Describe the feature

类似这样，很好很强大：https://github.com/PKU-YuanGroup/MoE-LLaVA ，一人血书QAQ

### Will you implement it?

- [ ] I would like to implement this feature and create a PR!

**Comment by ZwwWayne** [comment]
Thank you for your kind suggestion. The team is also studying this problem but it will take a long time to have one.
